lib/scrabble/scrabble.rb

```diff
@@ -0,0 +1,81 @@
+module Scrabble
+  class Scrabble
+
+    ONE_POINT = ["A", "E", "I", "O", "U", "L", "N", "R", "S", "T"]
+    TWO_POINTS = ["D", "G"]
+    THREE_POINTS = ["B", "C", "M", "P"]
+    FOUR_POINTS = ["F", "H", "V", "W", "Y"]
+    FIVE_POINTS = ["K"]
+    EIGHT_POINTS = ["J", "X"]
+    TEN_POINTS = ["Q", "Z"]
+
+    # RETURNS THE SCORE OF THE WORD
+    def self.score(word)
+      return 0 if valid?(word) == false
+      word.upcase!  # done so I don't need to .upcase every time
+
+      word_array = word.split(//)  # breaks word into an array of letters
+      points = 0
+
+      # Adds respective value to score if letter is included in the constant.
+      word_array.each do |letter|
```

Comment:
Ruby has a method on `String` named `each_char` which you could use instead of `split` followed by `each`. This is totally functional though!